concourse/pipelines/template/s3-br-upload-pipeline.yml.erb

```diff
@@ -30,16 +30,22 @@ resources:
     tag_filter: ((cf-ops-automation-tag-filter))
     skip_ssl_verification: true
 
-<% uniq_releases= {} %>
-<% all_dependencies.sort.each do |name,boshrelease| %>
-<% boshrelease["releases"]&.each do |release, info|  %>
-<% uniq_releases[release]= info %>
-<% end %>
-<% end %>
+<%
+  def bosh_io_hosted?(info)
```

Comment:
Maybe it is relevant to extract Info parsing to a dedicated class, rather than duplicating.